web/packages/selfhosted/test/polyfill/crossorigin_broken_cors/test.js

```diff
@@ -0,0 +1,17 @@
+const { open_test, inject_ruffle_and_wait } = require("../utils");
+const { expect, use } = require("chai");
+const chaiHtml = require("chai-html");
+
+use(chaiHtml);
+
+describe("Doesn't error with cross-origin frames", () => {
+    it("Loads the test", () => {
+        open_test(browser, __dirname);
+    });
+    it("Doesn't error", () => {
+        inject_ruffle_and_wait(browser);
+        const actual = browser.$("#test-container").getHTML(false);
```

Comment:
Looks like this should be `#test_container` instead of `#test-container`. Maybe this test should use mocha's [`throw`](https://www.chaijs.com/api/bdd/#method_throw), something like `expect(actual).to.not.throw()` ?

Reply:
I'm not familiar with `mocha`, if I remove the `window.error` code from `index.html` then add something like `expect(browser).to.not.throw` then the test should work?

As it is, the test should probably work unless I confused `-` & `_`, which is an easy fix.